### DISCUSSION TITLE
Problem on my Zephyrus with Arrow Keys

### DISCUSSION THREAD

**Discussion by Namelocked** [seerge/g-helper#2958] [Q&A] [answered]
Hey have some problems using my arrow keys with G-Helper.
Is there an Fix to use them correctly. Can just use them as Page up and Down and something like that, just not as arrow keys

**Comment by seerge** [comment, marked as answer]
@Namelocked hello. G-Helper is not handling and not touching your arrow keys anyhow :) 
Most likely you have changed something using some other apps.

**Comment by pdsway** [comment]
i have the same problem

killing G-Helper and/or all Asus services does not fix it.

My arrow keys can only do home/end and pgup/pgdn

FN key has no effect, FN-Lock no effect

**Reply by pdsway** [reply]
Solved...
- ASUS has an "arrow-lock" key next to the power button (however this did not work)
- I had "stopped" all Asus services
- Toggling FN-Lock and/or Arrow-Lock displayed the status on screen, but had no effect
- Restarted Asus services (G-Helper "start" button)
- Now the proper Arrow-Lock button works and regular arrow keys work

**Reply by RiverbackGames** [reply]
Thank you so much for your reply. I accidentally pressed the arrow-lock button and couldn't change it back. I tried to shut down the services but it didn't succesfully restart them afterwards and this is why this didn't work the first time. Second time around it worked. I installed armoury crate just to be safe.

Does the development crew of G-Helper know where this button does it magic? If (when) we lose the support from asus it would be nifty to manually fix this through registry or whatever. To be noted that this "arrow-lock" button changes the functionality of arrow keys on every single keyboard and also through remote connection. Also the actual functionality is faulty as I won't be able to use "SHIFT" and arrow keys to select text. When pressing shift the arrow keys functionality returns and vice versa if I lift SHIFT the alternate keys are active again. Uninstalling the keyboard through device manager even with driver does not help either.

It must be some kind of software layer and g-helper already accesses this partly with FN lock as the same UI is used when the arrow-lock button is used. There should be a command for that too. Maybe you can disable or alter this functionality as you have altered the function of the G button also? It seems that this software layer simply reads a keypress and if it's arrow press it translates it to one of the alternatives (Pg up, pg down, home, end). When I tried to press the arrow-lock button 100 times (While it was locked) and trying to use my arrow keys repeatedly I once or twice managed to use the arrow functionality of the keys which would prove my thought process above.

Anyhow, it's working now. This was driving me crazy. Thank you.
